### PR TITLE
fix eslint staff; add github actions

### PR DESCRIPTION
**Pull request by AndriiPolishko**

### CODE REVIEW

**Review by RiinaVi — APPROVED** [review]
LGTM!

File: .github/workflows/actions.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Code Quality Checks
```
Comment (skelemen-skelar) [inline .github/workflows/actions.yaml:1]: good job with additional task: pull_request + push, eslint + prettier
hope eslint and prettier will be never conflicting though)

File: .eslintrc.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ module.exports = {
   },
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
```
Comment (skelemen-skelar) [inline .eslintrc.js:12]: hmm... interesting... what's the point of using both?
plugin:@typescript-eslint/eslint-recommended
plugin:@typescript-eslint/recommended

**Review by skelemen-skelar — APPROVED** [review]
Good!

1. Branch was created
2. ESlint was installed. You have added some extensions - looks like they didn't change the code though)
2.1 Really went an extra mile with GHA actions (2 checks on 2 steps)
3. Linting works
4. Changes were pushed, PR was created

It would be interesting to see some additional rules for eslint besides presets and existing ones. Or maybe explanation of extensions you've added - maybe they do some really good stuff - I just don't see if they affected code for now.

P. S. change of prettier rule for quotes doesn't count))

**Pull request merged**